Index: app/src/main/java/com/example/beegame/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/beegame/MainActivity.java	(revision 8898b327af29c26f63d2247fdee64f0b2dfdefb1)
+++ app/src/main/java/com/example/beegame/MainActivity.java	(date 1585076039222)
@@ -2,44 +2,56 @@
 
 
 import android.os.Bundle;
+import android.view.View;
 import android.widget.ImageView;
 
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.constraintlayout.widget.ConstraintLayout;
 import androidx.constraintlayout.widget.ConstraintSet;
 
+import static androidx.constraintlayout.widget.ConstraintSet.BOTTOM;
+import static androidx.constraintlayout.widget.ConstraintSet.END;
+import static androidx.constraintlayout.widget.ConstraintSet.LEFT;
+import static androidx.constraintlayout.widget.ConstraintSet.PARENT_ID;
+import static androidx.constraintlayout.widget.ConstraintSet.TOP;
+
 
 public class MainActivity extends AppCompatActivity {
-
+    
     ConstraintLayout constraintLayout;
-    ConstraintSet set = new ConstraintSet();
-
-
+    
+    
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
         constraintLayout = new ConstraintLayout(this);
         constraintLayout = findViewById(R.id.constraint_layout);
-
+        
+        ConstraintSet set = new ConstraintSet();
+        
+        ImageView imageViewQueenBee = new ImageView(this);
+        int queenBeeIdView = View.generateViewId();
+        imageViewQueenBee.setId(queenBeeIdView);
+        constraintLayout.addView(imageViewQueenBee);
+        imageViewQueenBee.setImageResource(R.drawable.queen_bee);
+        set.connect(PARENT_ID, LEFT, queenBeeIdView, LEFT, 8);
+        set.connect(PARENT_ID, TOP, queenBeeIdView, TOP, 8);
+        
+        int prewIdView = queenBeeIdView;
         for (int i = 1; i <= 5; i++) {
             ImageView imageViewWorkerBee = new ImageView(this);
-            imageViewWorkerBee.setId(imageViewWorkerBee. generateViewId ());
+            int idThisView = View.generateViewId();
+            imageViewWorkerBee.setId(idThisView);
             constraintLayout.addView(imageViewWorkerBee);
             imageViewWorkerBee.setImageResource(R.drawable.worker_bee);
-            set.clone(constraintLayout);
-            set.connect(imageViewWorkerBee.getId(),ConstraintSet.LEFT, constraintLayout.NO_ID,
-                    ConstraintSet.END);
-
-
-        }
-
-
-
+            set.connect(prewIdView, LEFT, idThisView, LEFT);
+            set.connect(prewIdView, BOTTOM, idThisView, TOP);
+            
+            prewIdView = idThisView;
+            set.applyTo(constraintLayout);
+            
+            
+        }
     }
-
-  // private void changeConstraints(ConstraintSet set) {
-  //     set.connect(R.id.constraint_layout, ConstraintSet.LEFT,R.id.);
-  //     //set.connect(R.id.constraint_layout, ConstraintSet.BOTTOM, R.id.parent, ConstraintSet.TOP);
-  // }
 }
Index: .idea/dictionaries/Vitalii.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/dictionaries/Vitalii.xml	(date 1585074805877)
+++ .idea/dictionaries/Vitalii.xml	(date 1585074805877)
@@ -0,0 +1,3 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="Vitalii" />
+</component>
\ No newline at end of file
Index: .idea/codeStyles/codeStyleConfig.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/codeStyles/codeStyleConfig.xml	(date 1585074805847)
+++ .idea/codeStyles/codeStyleConfig.xml	(date 1585074805847)
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
+  </state>
+</component>
\ No newline at end of file
